Assets/SEE/Net/Network.cs

```diff
@@ -273,7 +256,30 @@ private void Awake()
         }
 
         /// <summary>
-        /// Makes sure that we have only one <see cref="Instance"/>.
+        /// Name of command-line argument for room password <see cref="RoomPassword"/>.
+        /// </summary>
+        private const string passwordArgument = "-password";
+
+        /// <summary>
+        /// Name of command-line argument for UDP port <see cref="ServerPort"/>.
```

Comment:
Just to check I understand this correctly: This is separate from the port running the HTTP API, right?